<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="GLOENML-GJUJG3Q" name="Android_1" tests="14" failures="1" timestamp="5 Oct 2016 11:35:11 GMT" time="77.047" errors="0">
  <testcase name="verify_tapon_myAccount" time="7.295" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_tapon_customerID" time="2.31" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_enter_customerID" time="7.328" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_tapon_continueButton" time="5.458" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_enter_userPassword" time="7.628" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_tocheckimageconfirmation" time="2.748" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_taponLoginbutton" time="7.075" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_DashboardpageRedirection" time="1.314" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_UserAccountNumber" time="5.335" classname="com.appModule.Signin_Test">
    <failure type="java.lang.AssertionError" message="expected [50100051926707] but found [50100051926708]">
      <![CDATA[java.lang.AssertionError: expected [50100051926707] but found [50100051926708]
	at com.appModule.Signin_Test.verify_UserAccountNumber(Signin_Test.java:100)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_UserAccountNumber -->
  <testcase name="verify_UserAccountBalance" time="4.336" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_FundTransferpageRedirection" time="0.872" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_tapon_NEFTTransaction" time="3.227" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_tapBeneficiaryDrpDownbox" time="5.374" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_BeneficiaryNames" time="2.016" classname="com.appModule.Signin_Test"/>
</testsuite> <!-- Android_1 -->
